README.md

```diff
@@ -1,70 +1,91 @@
-# Getting Started with Create React App
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/maosan132/calculator/">
+    <img src="./mLogo.png" alt="Logo" width="80" height="80">
+  </a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+  <h3 align="center">Calculator -
+  <a href='https://maosan132.github.io/calculator/'> Live version</a></h3>
```

Comment:
- [ ] It will be nice if you change the link of your `Live demo` to the correct one. In fact, this link doesn't display the same thing as your APP.

Reply:
Done, please note it is redirecting to this readme.